test/uk/gov/hmrc/cdsimportsddsfrontend/controllers/SubmitDeclarationControllerSpec.scala

```diff
@@ -74,7 +74,7 @@ class SubmitDeclarationControllerSpec extends CdsImportsSpec
     "when XML is valid" should {
       "Submit to the declaration service and return a success response" in signedInScenario { user =>
         val formData = Map("declaration-data" -> Seq("<declaration/>"))
-        val customsDeclarationsServiceMockSetup: CustomsDeclarationsService => Unit = ds => when(ds.submit(any(), any[Elem] )(any())).thenReturn(Future.successful(DeclarationServiceResponse("<foo></foo>", 200, Some("Good"))))
+        val customsDeclarationsServiceMockSetup: CustomsDeclarationsService => Unit = ds => when(ds.submit(any(), any[Elem], any())(any())).thenReturn(Future.successful(DeclarationServiceResponse("<foo></foo>", 200, Some("Good"))))
```

Comment:
I know you didn't create this pattern but it hides the interesting (when(...)  part off to the left. We've refactored some of these else where to be more readable so that the line starts with `when` and the reader can clearly see that is the Given part of the test